homeassistant/components/yamaha_musiccast/media_player.py

```diff
@@ -166,6 +166,27 @@ def state(self):
             return STATE_PLAYING
         return STATE_OFF
 
+    @property
+    def source_mapping(self):
+        """Return a mapping of the actual source names to their labels configured in the MusicCast App."""
+        ret = {}
+        for inp in self.coordinator.data.zones[self._zone_id].input_list:
```

Comment:
I'm not the biggest fan of the `inp` abbreviation here, but not really a big deal.